app/src/components/integrations/elements/integrationSettings/integrationSettings.jsx

```diff
@@ -47,6 +52,9 @@ export const IntegrationSettings = (props) => {
   const projectIntegrations = useSelector(namedProjectIntegrationsSelector);
   const projectId = useSelector(projectIdSelector);
   const activeProject = useSelector(activeProjectSelector);
+  const accountRole = useSelector(userAccountRoleSelector);
+  const userRole = useSelector(activeProjectRoleSelector);
+  const isAbleToEditByRole = canUpdateSettings(accountRole, userRole);
```

Comment:
I guess the behavior can be changed and the form will be disabled based not only on user role.
So me be name it f.e. as `isEditable` to be more general here.

Reply:
fixed ✓